src/market/api/api.ts

```diff
@@ -83,6 +83,7 @@ function sparklineAsSvgData(points: number[]): SparklineSvgData {
       })
       .join(" "),
     viewBox: `0 0 ${totalXSteps} ${sparklineYHeight + 3}`,
+    isPositive: points[0] <= points[points.length - 1],
```

Comment:
Can `points` be empty here? (noob question I have 0 context on market feature)

Reply:
good point should be handled now :+1: